sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -175,13 +175,13 @@ class HttpTransport(
 
     @abc.abstractmethod
     def send(self, request, **kwargs):
-        # type: (PipelineRequest, Any) -> PipelineResponse
+        # type: (PipelineRequest, Any) -> HTTPResponseType
         """Send the request using this HTTP sender.
 
         :param request: The pipeline request object
         :type request: ~azure.core.pipeline.PipelineRequest
```

Comment:
azure.core.pipeline.transport.HttpRequest

Reply:
The input is correct. It is ~azure.core.pipeline.PipelineRequest